src/mpvis/mpdfg/diagrammers/graphviz.py

```diff
@@ -207,5 +198,35 @@ def build_activity_name(self, activity_name: str) -> str:
 
         return activity_name
 
+    def get_arc_thickness_for_connection(self, connection):
```

Comment:
[REQ]

When instantiating `GraphVizDiagrammer` you define `self.arc_thickness_by` to `"frequency"` or `"time"`, so there is no need to check what the parameter value is. The function logic could be a lot cleaner doing something like:

```suggestion
    def get_arc_thickness_for_connection(self, connection):
        try:
            return link_width(
                    self.dfg["connections"][connection][self.arc_thickness_by],
                    self.connections_dimensions_min_and_max[self.arc_thickness_by],
                    )   
        except KeyError: # if by any chance self.arc_thickness_by value is not a key in self.dfg["connections"][connection] or self.connections_dimensions_min_and_max
            return 1
```